### ISSUE TITLE
syntax error

### ISSUE THREAD

**Issue by dylanmikesell** [iris-edu/piqqa#2]
Anyone having issues on a Mac -- intel chip.

I am trying to run an example after following the installation instructions.

`python /Users/dmikesell/GIT/piqqa/PIQQA.py --network=XP --start=2020-04-01 --end=2021-11-01` 

This is what I get. Cannot get past this line.

```
  File "/Users/dmikesell/GIT/piqqa/PIQQA.py", line 58
    print(f'         WARNING: Unable to retrieve values for requested channels, bypassing')
                                                                                         ^
SyntaxError: invalid syntax
```

**Comment by laura-iris** [comment]
Interesting, I'm not having the same problem when I run XP 2020-04-01 to 2021-11-01 on my Mac.  Can I ask if you created the conda environment using python3.9?  Older versions of python aren't compatible with the "f'" print formatting, so if it's running on and older version (prior to 3.6) then that could account for the syntax error.

**Comment by dylanmikesell** [comment]
I used 3.9 following the instructions on the GitHub page. I will double
check when back at my computer.

**Comment by dylanmikesell** [comment]
Sorry. This was my fault. Something was broken in anaconda path. I fixed it and am now using Python 3.9. Appears to be working just fine.
